components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementService.java

```diff
@@ -451,6 +453,18 @@ public abstract AuthenticationStep[] getConfiguredAuthenticators(String applicat
     public abstract AuthenticationStep[] getConfiguredAuthenticators(String applicationID)
             throws IdentityApplicationManagementException;
 
+    /**
+     * Get the list of trusted mobile applications based on the requested platform type.
+     *
+     * @param platformType Platform type of the trusted apps.
+     * @return List of trusted apps of all tenants.
+     * @throws IdentityApplicationManagementException If an error occurs while retrieving the trusted apps.
+     */
+    public List<TrustedApp> getTrustedApps(String platformType) throws IdentityApplicationManagementException {
```

Comment:
lets have this as an abstract method

Reply:
Thought of not introducing a new abstract method as there is a possibility of outsiders having custom implementations of this abstract class.